number_theory/numeric_unittest.cpp

```diff
@@ -0,0 +1,42 @@
+#include <stdint.h>
+
+#include <cmath>
+#include <limits>
+#include <random>
+
+#include <gtest/gtest.h>
+
+#include "number_theory/numeric.h"
+
+using namespace tql::number_theory;
+
+template <class T>
+void test_exgcd(T a, T b) {
+  auto [x, y] = exgcd(a, b);
+  if (a != 0 && b != 0) {
+    // Bézout's identity
+    ASSERT_LE(std::abs(x), std::abs(b));
+    ASSERT_LE(std::abs(y), std::abs(a));
+  }
+  ASSERT_EQ(x * a + y * b, gcd(a, b));
+}
+
+TEST(NumericTest, Exgcd) {
```

Comment:
I wonder if we are going to change the naming for GoogleTest. In our own files, we prefer using snake_case naming convention for our function names. This TEST macro creates a function with name `NumericTestExgcd` or something similar if I am not mistaken. Maybe, we should discuss how to name these parameters?